contracts/0.4.24/oracle/LidoOracle.sol

```diff
@@ -101,31 +101,32 @@ contract LidoOracle is ILidoOracle, AragonApp {
     So we should take another popular metric to express this - PPM or 1e-6
     daily_granularity_in_ppm = 2.74e-05 / 1e-6 = 27 (pretty good)
     with 100% buffer for future spikes it will be 50.
```

Comment:
Need to update comments, they are outdated.

Reply:
done
just left `If we use APR as a basic reference for increase, and expected range is between 1 and 10% APR.`

Reply:
`expected range is under 10% APR`